src/core_plugins/console/api_server/spec/overrides/indices.rollover.json

```diff
@@ -0,0 +1,12 @@
+{
+  "indices.rollover": {
+    "data_autocomplete_rules": {
+      "conditions" : {
+        "max_age": "7d",
+        "max_docs": 1000,
+        "max_size": "5gb"
+      },
+      "settings": { "__scope_link": "put_settings" }
```

Comment:
What does this do? How can we test this in the browser?

Reply:
I believe `__scope_link` lets autocomplete inherit from another set of suggestions. In this case, `settings` inherits `indices.put_settings`, defined by:
https://github.com/elastic/kibana/blob/52a7f4c1b6bcae8df445efffbcf75add07462b10/src/core_plugins/console/api_server/spec/generated/indices.put_settings.json#L1
and
https://github.com/elastic/kibana/blob/52a7f4c1b6bcae8df445efffbcf75add07462b10/src/core_plugins/console/api_server/spec/overrides/indices.put_settings.json#L1

So `settings` should suggest things like `number_of_replicas`, `analysis`, and further nested fields if any.

@bmcconaghy The documentation for rollover also says `mappings` and `aliases` can be part of the body so those should probably be added as well: https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-rollover-index.html#_defining_the_new_index